src/components/Lists/ListBase/ListBase.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import { Box as MaterialBox } from "@material-ui/core";
+import {
+    InboxRounded as InboxIcon,
+    StarBorderRounded as StarIcon,
+    SendOutlined as SendIcon,
+    DeleteOutline as DeleteIcon,
+    ReportOutlined as ReportIcon,
+    DraftsRounded as DraftsIcon,
+} from "@material-ui/icons";
+
+import ListBase, { ListBaseProps } from ".";
+import ListItemIconBase from "../../ListItemIcons/ListItemIconBase";
+import ListItemTextBase from "../../ListItemTexts/ListItemTextBase";
+import ListItemBase from "../../ListItems/ListItemBase";
+import TypographyBase from "../../Typography/TypographyBase";
+
+import { Story } from "@storybook/react";
+
+export default {
+    title: "Components/ListBase",
+    subcomponents: { ListItemBase, ListItemIconBase, ListItemTextBase, TypographyBase },
+    component: ListBase,
+};
+
+const simpleListItems = [
+    { icon: <StarIcon />, text: "Starred" },
+    { icon: <SendIcon />, text: "Sent" },
+    { icon: <DeleteIcon />, text: "Trash" },
+    { icon: <ReportIcon />, text: "Spam" },
+    { icon: <DraftsIcon />, text: "Drafts" },
+];
+
+export const SimpleList: Story<ListBaseProps> = (args) => (
+    <MaterialBox width={320}>
+        <ListBase {...args}>
+            <MaterialBox color="secondary.main" bgcolor="primary.main">
+                <ListItemBase>
+                    <MaterialBox ml={3}>

```

Comment:
ml should be determined by the list beforehand. 
If I were the user of this component, I expect to not write any styles whatsoever. Just variants
```suggestion
                    <MaterialBox}>

```